input/_partials/_elections-banner.cshtml

```diff
@@ -8,7 +8,7 @@
                 <p class="mx-auto">
                     This is your .NET Foundation! Get involved by voting for - or possibly running for - our next Board of Directors.
                 </p>
-                <div class="mx-auto d-flex justify-content-between site-button_row">
+                <div class="mx-auto justify-content-between site-button_row">
```

Comment:
Thanks. This all feels like black magic to me.

Reply:
Removed the flexbox layout so it could center as a single item; might want to add it back if you have 2 items (when it will fill the space between them)